src/services/db/GitFileSystemService.ts

```diff
@@ -1,5 +1,6 @@
 import fs from "fs"
 
+import { Base64 } from "js-base64"
```

Comment:
Oh yeah, do we still need this?

Reply:
Not in this PR at least, it might be needed for images? But if work needs to be done on that aspect we can tackle it in a separate PR! Removed in https://github.com/isomerpages/isomercms-backend/pull/892/commits/a4776f561cbe8392a1f208d940543c6406b867c3